### PR TITLE
[bitnami/common]: Address admission configuration typo 

### PR DESCRIPTION
**Pull request by odessosivan**
### Description of the change

As discussed on https://github.com/bitnami/charts/pull/19428, I noticed a typo in the new Pod Security Standard objects. This PR corrects the typo.

### Benefits

Using the new function will be typed correctly.

### Possible drawbacks

Nothing that I can think of

### Applicable issues

- Does not relate to any issue

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by juan131** [comment]
Draft PR looks fine, please feel free to promote it

File: bitnami/common/Chart.yaml [hunk 1]
```diff
@@ -23,4 +23,4 @@ name: common
 sources:
   - https://github.com/bitnami/charts
 type: library
-version: 2.13.2
```
Comment (jouve) [inline bitnami/common/Chart.yaml:26]: to have a backward compatible change, you need to keep the function with the typo, so both "common.capabilities.admisionConfiguration.supported" (adding a comment that it's deprecated) and "common.capabilities.admissionConfiguration.supported"

current change should bump to 3.0.0

ref: https://semver.org/

Comment (odessosivan) [reply bitnami/common/Chart.yaml:26]: Absolutely spot on.

Any objections to use the depreciated approach?

Comment (juan131) [reply bitnami/common/Chart.yaml:26]: You're right @jouve but we were quiet lucky this time and the macro hasn't been adopted by any chart in the catalog yet, see:
```bash
grep -Ri "common.capabilities.admisionConfiguration.supported" .
```
That's why I'd do an exemption this time.

Comment (odessosivan) [reply bitnami/common/Chart.yaml:26]: @juan131 @jouve

1. Can I consider this as resolved?
2. Do you need anything else from my side?

Comment (juan131) [reply bitnami/common/Chart.yaml:26]: Yes, let's consider it solved

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**